tests/e2e/fixtures/statsFixtures.js

```diff
@@ -462,6 +519,49 @@ const seasonTwoGames = [
     p1: 'player2',
   },
 ];
+const seasonFourGames = [
+  // Week 1
+  {
+    name: 'Hearts Week 1, Game 1',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs().subtract(1, 'week').valueOf(),
```

Comment:
The season started 2 weeks ago, so this should be 2 weeks ago to be in week 1
```suggestion
    updatedAt: dayjs().subtract(2, 'week').valueOf(),
```